2016-11-25-shiny-1.md

```diff
@@ -67,15 +67,15 @@ RStudio generates a template R script called `app.R`.
 
 Notice that the name given to your app was assigned to the directory, not the app script. For your app to work, the file must remain named `app.R`.
 
-It's possible to create a Shiny app with two files called `ui.R` and `server.R`, but the same can be accomplished by using one file. This tutorial will assume you have the one file app structure.
+It's possible to create a Shiny app with two files called `ui.R` and `server.R`, but the same can be accomplished by using one file. The difference between creating an app with two files and one file is... This tutorial will assume you have the one file app structure.
```

Comment:
I also made changes to lines 18 and 44, but accidentally merged them with the master branch, so you don't see them as new here, sorry!